src/app/(tabs)/event/page.tsx

```
@@ -13,6 +13,9 @@ export default function EventPage() {
   const [topTab, setTopTab] = useState<"신청 목록" | "내 이벤트" | "내 티켓">(
     "신청 목록",
   );
+  const [topTab, setTopTab] = useState<"신청 목록" | "내 이벤트" | "내 티켓">(
```

Biome (1.9.4)

[error] Shouldn't redeclare 'topTab'. Consider to delete it or rename it.

'topTab' is defined here:


(lint/suspicious/noRedeclare)